build.sbt

```diff
@@ -500,6 +500,18 @@ lazy val jettyClient = libraryProject("jetty-client")
   )
   .dependsOn(core, testing % "test->test", client % "compile;test->test")
 
+lazy val nodeServerless = libraryProject("node-serverless", CrossType.Pure, List(JSPlatform))
```

Comment:
what is this?? 

I am subscribed to your PR so getting notifications on every commit, this is incredible.

![tenor](https://user-images.githubusercontent.com/2464813/127752718-c9c754cd-6029-4fe0-9689-2794eea2a64c.gif)

Reply:
Yup, this is where this where we were headed all along :)